pandas/util/_decorators.py

```diff
@@ -267,7 +267,7 @@ def deprecate_nonkeyword_arguments(
 
     Parameters
     ----------
-    version : str
+    version : str, optional
```

Comment:
maybe could set None as the default

Reply:
True, but to be fair to them I didn't write that in the issue...IMO might as well merge as-in, the function itself could be given a default value later